Lecture-8/Task-8.1.py

```diff
@@ -0,0 +1,13 @@
+user_word = input("შეიყვანეთ შესამოწმებელი სიტყვა: ").lower()
+
+user_letters = []
+
+for letter in user_word:
+    if "{" > letter > "`":
+        user_letters.append(letter)
+        print(user_letters)
+
+if user_letters == list(reversed(user_letters)):
+    print("Is a palindrome")
+else:
+    print("Is not a palindrome")
```

Comment:
გთავაზობ ლისტის გარეშე უფრო მარტივ ამონახსნს:
input_word = input ("Please enter a string and you will see if it is a palindrome: ")
# მხოლოდ ასოებს გაითვალისწინებ და ამის მიხედვით დაადგენს პალინდრომობას
only_characters = ""
for i in input_word:
    if i.isalpha():
        only_characters += i

only_characters = only_characters.lower()

reversed_word = ""

for i in range (len(only_characters)-1,-1,-1):
    reversed_word += only_characters[i]
print (f"Reversed: {reversed_word}")
if reversed_word == only_characters:
    print(f"\"{input_word}\" is a palindrome")
else:
    print(f"\"{input_word}\" is not a palindrome")